assets/scss/templates/_signup.scss

```diff
@@ -124,28 +119,21 @@
   }
 }
 @include media-breakpoint('desktop') {
-  .feature-panel {
-    display: none;
-  }
-}
-@include media-breakpoint('mobile') {
   .panel-wrapper {
     .main-panel {
-      .signup-form {
-        .holder-small {
-          width: 100%;
-          margin-right: 0;
-          display: block;
-
-          &:nth-child(1) {
-            margin-right: 0;
-          }
-        }
+      width: 100%;
 
-        ul > li:nth-child(n+2) {
-          margin-top: 20px;
+      .field-item--50 {
+        width: 100%;
+        // Left Column
+        &:nth-of-type(2n+1) {
+          padding-right: 0;
```

Comment:
fix still pending, right?

Reply:
Yes, this was already corrected in the next commit

Reply:
Perdón, no entendí, _fue_ corregido o _va a ser_ corregido? en el último caso, dentro de este mismo pr o en otro?

Reply:
Si, ya lo corregí en este pull request, mas a bajo vas a ver el commit

Reply:
En ese caso, la variable `$dp-spaces--lv0` no tiene sentido, deberíamos borrarla?

https://github.com/FromDoppler/Doppler-UI-System/pull/59/files#diff-c6347520a93a62f1f778dd66f6bba5d3R42

@GusBaamonde @mvillaseco

Reply:
Seguramente la necesitaremos, sobre todo si por herencia los elementos generan margins o paddings, esta variable nos ayudaría de mucho.

Reply:
Ok, entonces la agrego ;)